doc/apimanager-reference.md

```diff
@@ -198,6 +199,8 @@ Only one of the fields can be chosen. If no field is specified then PVC is used.
 | **Field** | **json/yaml field**| **Type** | **Required** | **Default value** | **Description** |
 | --- | --- | --- | --- | --- | --- |
 | StorageClassName | `storageClassName` | string | No | nil | The Storage Class to be used by the PVC |
+| Resources | `resources` | [PersistentVolumeClaimResourcesSpec](#PersistentVolumeClaimResourcesSpec) | No | nil | The minimum resources the volume should have. Resources will not take any effect when VolumeName is provided. |
```

Comment:
:+1: about not taking effect when VolumeName is provided.

Could we document here also that the parameter is not updateable when the underlying PV is not resizable? (see comment https://github.com/3scale/3scale-operator/pull/467#issuecomment-700603059)

Can we also add somewhere (maybe in the highAvailability attribute) the information about internal DBs not being intended for production environments?